### DISCUSSION TITLE
Capcom CPS1/CPS2/CPS3 emulator for arcade games.

### DISCUSSION THREAD

**Discussion by YEEHAW007** [QuiteAFancyEmerald/InvisiProxy#170] [Ideas]
I really just want to play the Street fighter games and Final Fight. For the Retroarch core I suggest Final Burn Neo.

**Comment by foxsouns** [comment]
fork [webretro](https://github.com/BinBashBanana/webretro) and find your core's files, copy them to root, and selfhost, use example url endings given in readme
